pipelines/scipipe/ap_verify.groovy

```diff
@@ -444,6 +467,7 @@ def void runApVerify(Map p) {
     "DATASET_DIR=${p.datasetDir}",
     "HOME=${p.homeDir}",
     "CODE_DIR=${p.codeDir}",
+    "RBCLASSIFIER_DATA_DIR=${p.rbClassifierDataDir}",
```

Comment:
Added `// TODO: remove the below line after DM-38454` comment on line 470 and shifted the variable assignment since comment on same line exceeded the total length of 79.